llvm/test/lab3/kalinin_alexandr/test.mir

```diff
@@ -0,0 +1,252 @@
+# RUN: llc -mtriple x86_64-unknown-linux-gnu --load=%llvmshlibdir/X86SKalininMICounterPass%shlibext -run-pass=x86-kalinin-mi-counter  %s -o - | FileCheck %s
+
+# test.cpp
+#
+# unsigned long ic;
```

Comment:
Add check for the case when there is no "ic" to cover the situation when you create "ic" by yourself

Reply:
Sorry, but Mikhail Mecheslavovich said that according to the conditions of the problem, the variable ic exists. Still, do we need to display this case too?

Reply:
In this case there is no need to create global variable in the pass

Reply:
Sorry, as a result, I need to redo the tests so that there is no ic variable?

Reply:
No-no. Just need to remove if statement where you create "ic" global ver. That's it

Reply:
ok, I understand you